ClientTests/NTPDownloaderTests.swift

```diff
@@ -86,6 +103,114 @@ class NTPDownloaderTests: XCTestCase {
             ]
         }
         """
+
```

Comment:
@tmancey It would be good to have test for wallpaper json that includes `wallpapers` and `campaigns` data both.
According to the docs (https://docs.google.com/document/d/1unuYlJXawx73Y5K6b54soS5HAALAwDQ1RS86AkNQHb0/edit#heading=h.bgp0ht46fee2), json could have both.
And, if this json is provided, old browser will pick `wallpapers` and new browser will pick `campaigns`?

Reply:
Added a test, thanks